app/models/concerns/exportable.rb

```diff
@@ -97,34 +116,18 @@ def reached_patient_call_count
   end
 
   def has_alt_contact?
-    other_contact.present? || other_phone.present? || other_contact_relationship.present?
+    if is_a?(Patient)
+      determine_has_other_contact
```

Comment:
I think this might confuse me a bit if we rely on the parent classes for this. I think it's alright if we keep the logic for this in here rather than delegating it back to the classes it's attached to.

e.g.

```
if is_a? Patient
  thing || other thing ||
else
  whatever the archived pt behavior is
```

Interested in your thoughts!

Reply:
renamed the function on the Patient to `has_alt_contact` and removed the function from here completely. Now the call inside the export proper calls the correct one.